docker/ckan/2.9.3.Dockerfile

```diff
@@ -0,0 +1,121 @@
+# See CKAN docs on installation from Docker Compose on usage
+FROM governmentdigitalservice/ckan-python:3.6.12 AS base
+MAINTAINER Open Knowledge
+
+# Set timezone
+ENV TZ=UTC
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Setting the locale
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+# Install required system packages
+RUN apt-get -q -y update \
+    && DEBIAN_FRONTEND=noninteractive apt-get -q -y upgrade \
+    && apt-get -q -y install \
+        python3-dev \
+        python3-pip \
+        python3-wheel \
+        libpq-dev \
+        libxml2-dev \
+        libxslt-dev \
+        libgeos-dev \
+        libssl-dev \
+        libffi-dev \
+        postgresql-client \
+        build-essential \
+        git-core \
+        vim \
+        wget \
+        curl \
+        proj-bin \
+        libproj-dev \
+    && apt-get -q clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Define environment variables
+ENV CKAN_HOME /usr/lib/ckan
+ENV CKAN_VENV $CKAN_HOME/venv
+ENV CKAN_CONFIG /etc/ckan
+ENV CKAN_STORAGE_PATH=/var/lib/ckan
+
+# Create ckan user
+RUN useradd -r -u 900 -m -c "ckan account" -d $CKAN_HOME -s /bin/false ckan
+
+# Setup virtual environment for CKAN
+RUN mkdir -p $CKAN_VENV $CKAN_CONFIG $CKAN_STORAGE_PATH && \
+    python3.6 -m venv $CKAN_VENV && \
+    ln -s $CKAN_VENV/bin/ckan /usr/local/bin/ckan
+
+# Virtual environment binaries/scripts to be used first
+ENV PATH=${CKAN_VENV}/bin:${PATH}  
+
+# copy production.ini
+ADD . $CKAN_VENV/src/ckanext-datagovuk/
```

Comment:
For copying from the host or another container (as opposed to downloading a URL), `COPY` is preferable to `ADD` because it has fewer surprising features / gotchas.

Reply:
ok, will fix that